packages/react-router-dev/config/routes.ts

```diff
@@ -92,11 +92,6 @@ export interface RouteConfigEntry {
 
 type CreateRoutePath = string | null | undefined;
 
-type RequireAtLeastOne<T> = {
```

Comment:
I thought that this was causing type issues when trying to wrap the route helpers. This turned out not to be the case, but in either case it prompted me to revisit this and I felt like this level of type checking was overkill.